src/main/java/thePackmaster/cards/startuppack/Cross.java

```diff
@@ -0,0 +1,63 @@
+package thePackmaster.cards.startuppack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDrawPileAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.powers.startuppack.CrossPower;
+import thePackmaster.util.Wiz;
+
+import java.util.Objects;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Cross extends AbstractStartUpCard {
+    public final static String ID = makeID("Cross");
+    public Cross() {
+        super(ID, 1, CardType.ATTACK, CardRarity.UNCOMMON, CardTarget.ENEMY);
+        this.baseDamage = this.damage = 7;
+        this.baseSecondDamage = this.secondDamage = 0;
+    }
+
+    @Override
+    public void use(AbstractPlayer abstractPlayer, AbstractMonster abstractMonster) {
+        addToBot(new DamageAction(abstractMonster, new DamageInfo(abstractPlayer, this.damage, DamageInfo.DamageType.NORMAL), AbstractGameAction.AttackEffect.SLASH_HORIZONTAL));
+        addToBot(new DamageAction(abstractMonster, new DamageInfo(abstractMonster, this.secondDamage, DamageInfo.DamageType.NORMAL), AbstractGameAction.AttackEffect.SLASH_VERTICAL));
+        AbstractPackmasterCard cross = new Cross();
+        Wiz.applyToSelf(new CrossPower(abstractPlayer, 1));
+        if(this.upgraded){
+            cross.upgrade();
+        }
+        addToBot(new MakeTempCardInDrawPileAction(cross, 1, true, false));
+    }
+
+    public void applyPowers() {
+        this.baseSecondDamage = 0;
+        for(AbstractPower p : AbstractDungeon.player.powers){
+            if(Objects.equals(p.ID, makeID("CrossPower"))){
```

Comment:
```suggestion
            if(CrossPower.POWER_ID.equals(p.ID)){
```
You were passing a string literal with extra steps, make sure to use the static id field every power has. This also adds null safety and is easier to read